src/assets/js/diy.js

```diff
@@ -26,53 +26,60 @@
                 },
                 {
-                    'name': 'BNO055',
+                    'name': 'ICM20948',
                     'amount': (set) => set,
-                    'cost': 55.10,
-                    'costAll': (set) => set * 55.10 + 2.73,
-                    'links': '<a href="https://www.aliexpress.com/wholesale?SearchText=BNO055">AliExpress BNO055</a>'
+                    'cost': 17.40,
+                    'costAll': (set) => set * 17.40 + 8.89,
+                    'links': '<a href="https://www.mouser.com/c/?q=ICM20948">Mouser ICM20948</a>. The most commonly chosen options are either the Pimoroni or Adafruit ICM20948. Please note that any orders you place will be <b>backordered</b>, potentially meaning a wait of <u>upwards of 3 months</u> before shipping.'
                 },
                 {
-                    'name': 'BMI160',
+                    'name': 'MPU9250',
                     'amount': (set) => set,
-                    'cost': 1.42,
-                    'costAll': (set) => set * 1.42 + 2.67,
-                    'links': '<a href="https://www.aliexpress.com/wholesale?SearchText=BMI160">AliExpress BMI160</a>'
+                    'cost': 4.75,
+                    'costAll': (set) => set * 4.75 + 2.73,
+                    'links': '<a href="https://www.aliexpress.com/wholesale?SearchText=MPU9250">AliExpress MPU9250</a>. Buyer beware: large amount of fakes. <b>Not recommended for new designs.</b>'
                 },
                 {
                     'name': 'MPU+QMC5883L',
                     'amount': (set) => set,
                     'cost': 1.04+1.23,
                     'costAll': (set) => (set * (1.04+1.23)) + 2.67,
-                    'links': '<a href="https://www.aliexpress.com/wholesale?SearchText=MPU6050">AliExpress MPU6050</a> and <a href="https://www.aliexpress.com/wholesale?SearchText=QMC5883L">AliExpress QMC5883L</a>. Performance should apprxomiately match an MPU9250, but please note that this is <b>Experimental</b>.'
+                    'links': '<a href="https://www.aliexpress.com/wholesale?SearchText=MPU6050">AliExpress MPU6050</a> and <a href="https://www.aliexpress.com/wholesale?SearchText=QMC5883L">AliExpress QMC5883L</a>. Performance should apprxomiately match an MPU9250, but please note that this is <b>Experimental</b>. <b>Not recommended for new designs.</b>'
                 },
                 {
-                    'name': 'MPU9250',
+                    'name': 'BNO055',
                     'amount': (set) => set,
-                    'cost': 4.75,
-                    'costAll': (set) => set * 4.75 + 2.73,
-                    'links': '<a href="https://www.aliexpress.com/wholesale?SearchText=MPU9250">AliExpress MPU9250</a>. Buyer beware: large amount of fakes.'
+                    'cost': 55.10,
+                    'costAll': (set) => set * 55.10 + 2.73,
```

Comment:
This definitely isn't accurate anymore if you are looking to update more :P

Reply:
I set it to $17, which is a good average of ali prices I think. Not that anyone should use these :P